stage1/init/mount.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2014 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// file with functions that allow for creating a mount units for
+// managing inner and external volumes in stage1
+// mount units from systemd require to have /usr/bin/mount
+// TODO: it's would better to just use host@.mount empty@.mount systemd templates, but
```

Comment:
Does this technique requires systemd 222 on host or in stage1? If in the latter only then please consider implementing it - we bumped systemd version in coreos to 222 recently (#1077).